File: HorseRacing.Contracts/Models/User/Responses/GetUser/GetUserResponseDto.cs
using HorseRacing.Contracts.Base.Dto;

namespace HorseRacing.Contracts.Models.User.Responses.GetUser
{
    public class GetUserResponseDto : BaseDto
    {
        /// <summary>
		/// Имя
		/// </summary>
		public string FirstName { get; set; } = "";
        /// <summary>
        /// Фамилия
        /// </summary>
        public string LastName { get; set; } = "";
        /// <summary>
        /// Фамилия
        /// </summary>
        public string UserName { get; set; } = "";
        /// <summary>
		/// Электронная почта
		/// </summary>
		public string Email { get; set; } = "";
        /// <summary>
        /// Телефон
        /// </summary>
        public string Phone { get; set; } = "";
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/user/responses/get-user/get-user-response-dto.ts
import type { BaseDto } from '~/interfaces/api/contracts/base/dto/base-dto';

export interface GetUserResponseDto extends BaseDto {
    /** Имя */
    FirstName: string;
    /** Фамилия */
    LastName: string;
    /** Логин */
    UserName: string;
    /** Телефон */
    Phone: string;
    /** Почта */
    Email: string;
}
